config/generate-templates.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
I don't know if it's copied/pasted or generated but it's time to update years )

Reply:
I don't know where I copy-pasted from :)

Reply:
Fixed all license years